packages/react/src/isomorphicClerk.ts

```diff
@@ -198,6 +198,8 @@ export default class IsomorphicClerk {
         await global.Clerk.load(this.options);
       }
 
+      global.Clerk.sdkMetadata = this.options.sdkMetadata ?? { name: PACKAGE_NAME, version: PACKAGE_VERSION };
```

Comment:
For SDKs that use `ClerkProvider`, this is how we are passing through and setting the metadata. SDKs that access `Clerk` directly assign it themselves.